pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -71,14 +73,35 @@ func (ds *dockerService) determinePodIPBySandboxID(sandboxID string) string {
 		if err != nil {
 			continue
 		}
-		if containerIP := getContainerIP(r); containerIP != "" {
-			return containerIP
+
+		// On Windows, every container that is created in a Sandbox, needs to invoke CNI plugin again for adding the Network,
+		// with the shared container name as NetNS info,
+		// which is passed down the platform to replicate some necessary changes on to the new container
+		//
+		// This place is chosen as a hack for now, until we have a better place to do this,
+		// like immediately after ContainerCreation,
+		// Also this is very windows specific for now
```

Comment:
remove `Also this is very windows specific for now`
I think it's pretty clear from the context that this is windows specific.